src/AccountLabel.jsx

```diff
@@ -58,54 +60,59 @@ class AccountLabelAux extends ReactiveComponent {
 	readyRender () {
 		let badges = this.state.badges.map((b, i) => (
 			<Rimg
-				key={i}
-				alt={b.caption}
-				src={bonds.githubhint.entries(b.img)[0]}
-				style={{
+				key={ i }
+				alt={ b.caption }
+				src={ bonds.githubhint.entries(b.img)[0] }
+				style={ {
 					marginLeft: '0.25em',
 					marginRight: '0.25em',
 					border: '6px solid transparent'
-				}}
+				} }
 			/>
 		));
 
 		return (
 			<Label
 				as='a'
-				color={this.state.names.owned ? 'yellow' : 'blue'}
+				color={ this.state.names.owned ? 'yellow' : 'blue' }
 				image
-				basic={!(this.state.names.owned || this.state.names.registry)}
-				style={{
+				basic={ !(this.state.names.owned || this.state.names.registry) }
+				style={ {
 					borderTopLeftRadius: this.props.noicon ? 'default' : '16px',
 					borderBottomLeftRadius: this.props.noicon ? 'default' : '16px',
 					overflow: 'hidden',
 					verticalAlign: 'sub'
-				}}
+				} }
 			>
-				{this.props.noicon ? '' : (<AccountIcon
-					address={this.props.address}
-					style={{
-						borderRadius: '50%',
-						border: '2px solid transparent'
-					}}
-				/>)}
+				{this.props.noicon
+					? ''
+					: (
+						<AccountIcon
+							address={ this.props.address }
+							style={ {
+								borderRadius: '50%',
+								border: '2px solid transparent'
+							} }
+						/>)
+				}
 				{this.state.names.owned || this.state.names.registry || (
 					<span>
-						<span style={{
+						<span style={ {
 							fontSize: 'x-small',
 							fontWeight: 'lighter'
-						}}>
+						} }
+						>
 							0x
 						</span>
 						{this.props.address.substr(2, 8)}…{this.props.address.slice(-4)}
 					</span>
 				)}
-				{(badges.length > 0 || this.state.names.owned || this.state.names.registry) ? (
-					<Label.Detail style={{paddingLeft: 0, paddingRight: 0}}>
-					{badges}
-					</Label.Detail>
-				) : ''}
-		    </Label>
+				{(badges.length > 0 || this.state.names.owned || this.state.names.registry) &&
+					(<Label.Detail style={ { paddingLeft: 0, paddingRight: 0 } }>
+						{badges}
+					</Label.Detail>)
+				}
+			</Label>
 		);
 	}
 }
```

Comment:
Add propTypes for every file. It's required by best practices. Add defaultProps only where it's relevant.